src/gui/folder.cpp

```diff
@@ -1000,17 +932,9 @@ void Folder::startSync()
 
     setDirtyNetworkLimits();
 
-    static std::chrono::milliseconds fullLocalDiscoveryInterval = []() {
-        auto interval = ConfigFile().fullLocalDiscoveryInterval();
-        QByteArray env = qgetenv("OWNCLOUD_FULL_LOCAL_DISCOVERY_INTERVAL");
-        if (!env.isEmpty()) {
-            interval = std::chrono::milliseconds(env.toLongLong());
-        }
-        return interval;
-    }();
-    bool hasDoneFullLocalDiscovery = _timeSinceLastFullLocalDiscovery.isValid();
-    bool periodicFullLocalDiscoveryNow =
-        fullLocalDiscoveryInterval.count() >= 0 // negative means we don't require periodic full runs
+    const std::chrono::milliseconds fullLocalDiscoveryInterval = ConfigFile().fullLocalDiscoveryInterval();
+    const bool hasDoneFullLocalDiscovery = _timeSinceLastFullLocalDiscovery.isValid();
+    const bool periodicFullLocalDiscoveryNow = fullLocalDiscoveryInterval.count() >= 0 // negative means we don't require periodic full runs
```

Comment:
Please avoid inline comments. In this case especially, the line has become quite long.

Reply:
this is old code but well I can format it to your liking